pkg/config/config.go

```diff
@@ -11,11 +11,12 @@ import (
 
 // Serialized defines the standard config for this operator.
 type Serialized struct {
-	Report      bool   `json:"report"`
-	StoragePath string `json:"storagePath"`
-	Interval    string `json:"interval"`
-	Endpoint    string `json:"endpoint"`
-	PullReport  struct {
+	Report                      bool   `json:"report"`
+	StoragePath                 string `json:"storagePath"`
+	Interval                    string `json:"interval"`
+	Endpoint                    string `json:"endpoint"`
+	ConditionalGathererEndpoint string `json:"conditionalGathererEndpoint"`
```

Comment:
Just suggestion. What about `conditionsEndpoint`?

Reply:
the idea was that we may define more in the json later (it's likely to not happen tho) and term `conditions` could be confused with operator conditions